jsLearn/logicalOperators.js

```diff
@@ -0,0 +1,28 @@
+// 1.
+
+let age = prompt("write age", 20);
+let result = age >= 14 && age <= 90 ? true : false;
+
+
+// 2
+
+let age = prompt("write age", 20);
+let result = !(age >= 14 && age <= 90) ? true : false;
+let result1 = age < 14 && age >90 ? true : false;
```

Comment:
both your conditions return a boolean value, no need for ternary operator

Reply:
done